### PR TITLE
[157][166] 0.4.0 docs

### PR DESCRIPTION
**Pull request by jkbradley**
Updated for release 0.4.0

See https://github.com/graphframes/graphframes/pull/180 for a list of changes made to generate these docs.  Other than those changes, I generated these as follows:
* Remove "SNAPSHOT" suffix in build.sbt
* Clean sbt and doc build
  * Doc build was run with PRODUCTION=1
* Copied files in graphframes/docs/_site to this repo.

### CODE REVIEW

**Comment by jkbradley** [comment]
@thunterdb  I'm a little confused by how much JS, CSS, etc. this is removing.  Do you think it's because of a different Jekyll version or something?

File: api/python/genindex.html [hunk 1]
```diff
@@ -291,13 +291,13 @@ <h2 id="V">V</h2>
...
-        <li class="nav-item nav-item-0"><a href="index.html">pygraphframes 0.2.0 documentation</a> &#187;</li>
+        <li class="nav-item nav-item-0"><a href="index.html">pygraphframes 0.4.0 documentation</a> &raquo;</li>
  
       </ul>
     </div>
     <div class="footer" role="contentinfo">
-        &#169; Copyright 2016, Timothy Hunter and Joseph Bradley.
-      Created using <a href="http://sphinx-doc.org/">Sphinx</a> 1.4.6.
+        &copy; Copyright 2016, Timothy Hunter and Joseph Bradley.
+      Created using <a href="http://sphinx-doc.org/">Sphinx</a> 1.4.1.
```
Comment (felixcheung) [inline api/python/genindex.html]: older sphinx version?

**Comment by jkbradley** [comment]
So I updated this with the latest sphinx, and now there are a lot of additions instead of deletions.  I did fix one thing: This PR's docs have corrected links to Python source code.  Those links are broken in the current docs.

File: user-guide.html [hunk 2]
```diff
@@ -29,6 +29,19 @@
         <link rel="stylesheet" href="css/pygments-default.css">
 
         
+        <!-- Google analytics script -->
+        <script type="text/javascript">
```
Comment (felixcheung) [inline user-guide.html]: this is actually being tracked? (not a problem, just want to confirm)

Comment (jkbradley) [inline user-guide.html]: Hm, the current page does not, but I think we originally did intend to track interest.  I bet the current docs weren't generated correctly.

**Comment by jkbradley** [comment]
Alright, here goes nothing.  Merging.

Thanks @felixcheung !

**Comment by jkbradley** [comment]
Well that screwed up the python API docs...though they worked locally.  Will try to fix now.

**Comment by jkbradley** [comment]
I don't see anything wrong with the page source.  I'll wait to see if it just takes time for all of the files to be available.

**Comment by jkbradley** [comment]
This is really weird.  Github.io has pushed some of the new files but not all.

**Comment by jkbradley** [comment]
Success!  added .nojekyll file in next PR

**Pull request merged**